### DISCUSSION TITLE
Halmat Analog Engine oil pressure code help?

### DISCUSSION THREAD

**Discussion by addboy58** [hatlabs/discussions#126] [Q&A]
This is my first time working with code like this. Below is my try at modifying the example code to read the oil pressure from an analogue oil pressure sender. It works off of resistance. Would like to know if it is right.

(:/NEMA 2000/emgine 1 dynamic update", 0 nema200); ,, engine1, instance 0

Oil a2 Frequency->conect_to(&(engine_dynamic_sender->engineoil.pressure_consumer_));

Thx
in advance
Dave

**Comment by mairas** [comment]
Your code copy paste might've failed in some interesting ways. It's totally garbled.

**Comment by addboy58** [comment]
Ok, point taken. So how would you do it?

**Comment by mairas** [comment]
Just a regular copy-paste as a code block: separated by ``` markers on their own lines before and after the code.
